### ISSUE TITLE
check errors, test predict hangs

### ISSUE THREAD

**Issue by ruy-jauregui** [nextgenusfs/funannotate#823]
#I have installed the Conda package version 1.8.13
#check throws these errors:
ruy@inv:/dataset/funannotate$ funannotate check
-------------------------------------------------------
Checking dependencies for 1.8.13
-------------------------------------------------------
To print all dependencies and versions: funannotate check --show-versions

You are running Python v 3.8.13. Now checking python packages...
All 11 python packages installed

You are running Perl v b'5.032001'. Now checking perl modules...
All 27 Perl modules installed

Checking Environmental Variables...
        ERROR: GENEMARK_PATH not set. export GENEMARK_PATH=/path/to/dir
-------------------------------------------------------
Checking external dependencies...
pigz 2.3.4
        ERROR: emapper.py not installed
        ERROR: gmes_petap.pl not installed
        ERROR: pigz not installed

# And yet, pigz is there. I don't know where to look for the missing Python and Perl scripts.
ruy@inv:/dataset/funannotate$ which pigz
/usr/bin/pigz

#  funannotate test -t predict hangs here:

#########################################################
Running `funannotate predict` unit testing
CMD: funannotate predict -i test.softmasked.fa --protein_evidence protein.evidence.fasta -o annotate --augustus_species saccharomyces --cpus 2 --species Awesome testicus
#########################################################
-------------------------------------------------------
[Oct 26 02:10 PM]: OS: CentOS Linux 7, 24 cores, ~ 396 GB RAM. Python: 3.8.13
[Oct 26 02:10 PM]: Running funannotate v1.8.13
[Oct 26 02:10 PM]: GeneMark not found and $GENEMARK_PATH environmental variable missing. Will skip GeneMark ab-initio prediction.

# nothing happens after days waiting, assigning more threads doesn't change this.

Any help is greatly appreciated

**Comment by hyphaltip** [comment]
Yeah this is reported earlier. Working on test fix. What version of Augustus installed ?

**Comment by ruy-jauregui** [comment]
The bundled Augustus version on the Conda package is 3.5.0

**Comment by nextgenusfs** [comment]
Please upgrade to latest in master which I think the problem is fixed, its just not in a tagged release yet.

**Comment by ruy-jauregui** [comment]
Thanks. The latest master  throws an error on the matplotlib version:
/dataset/bioinformatics_dev/itmp/jaureguir/funannotate-011122/funannotate-master$ python3.10 setup.py install --prefix /dataset/bioinformatics_dev/itmp/jaureguir/funannotate-011122
...
Installed /bifo/itmp/bioinformatics_dev/jaureguir/funannotate-011122/lib/python3.10/site-packages/contourpy-1.0.6-py3.10-linux-x86_64.egg
error: matplotlib 3.6.1 is installed but matplotlib!=3.6.1,>=3.1 is required by {'seaborn'}

**Comment by nextgenusfs** [comment]
python -m pip install .... --upgrade --force --no-deps

**Comment by ruy-jauregui** [comment]
Sorry, unless I'm not catching up, It doesn't help.
jaureguir@invbfopp10:/dataset/bioinformatics_dev/itmp/jaureguir/funannotate-011122/funannotate-master$ python3.10 -m pip install matplotlib  --upgrade --force --no-deps
Collecting matplotlib
  Using cached matplotlib-3.6.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (11.8 MB)
Installing collected packages: matplotlib
  Attempting uninstall: matplotlib
    Found existing installation: matplotlib 3.6.1
    Uninstalling matplotlib-3.6.1:
      Successfully uninstalled matplotlib-3.6.1
Successfully installed matplotlib-3.6.1
this is the same version that throws the error. The error stays the same:
error: matplotlib 3.6.1 is installed but matplotlib!=3.6.1,>=3.1 is required by {'seaborn'}

**Comment by ruy-jauregui** [comment]
Further confusion: 
This seems to run OK:
python3.10 -m pip install matplotlib==3.6.0 --force --no-deps
#and yet the error stays the same...
I even tried installing matplotlib 3.6.0 as a conda environment, but the error stays.
